master/networking/advertise-service-ips.md

```diff
@@ -98,6 +98,11 @@ If your {{site.prodname}} deployment is configured to peer with BGP routers outs
 
    For help see, [BGP configuration resource]({{site.baseurl}}/{{page.version}}/reference/resources/bgpconfig).
 
+> **Note**: If you are setting a cluster CIDR using the legacy environment variable CALICO_ADVERTISE_CLUSTER_IPS,
```

Comment:
So, I think this might be a bit confusing in this doc, since we don't actually talk about this env var anywhere except in this note. WDYT?

Reply:
WDYT this edit:

```
In Calico versions < v3.10, service cluster IP advertisement was configured via the environment variable CALICO_ADVERTISE_CLUSTER_IPS.
The environment variable takes precedence over any serviceClusterIPs configured in the default BGPConfiguration. We recommend replacing the
deprecated CALICO_ADVERTISE_CLUSTER_IPS with BGPConfiguration.
```